internal/services/conditionalaccess/conditional_access_policy_resource.go

```diff
@@ -397,6 +397,15 @@ func conditionalAccessPolicyResource() *schema.Resource {
 							}, false),
 						},
 
+						"persistent_browser_mode": {
+							Type:     schema.TypeString,
+							Optional: true,
+							ValidateFunc: validation.StringInSlice([]string{
+								msgraph.PersistentBrowserSessionModeAlways,
+								msgraph.PersistentBrowserSessionModeNever,
+							}, false),
+						},
```

Comment:
this seems to be a boolean so could become a bool? maybe
```suggestion
						"persistent_browser_cookies_enabled": {
							Type:     schema.TypeString,
							Optional: true,
							ValidateFunc: validation.StringInSlice([]string{
								msgraph.PersistentBrowserSessionModeAlways,
								msgraph.PersistentBrowserSessionModeNever,
							}, false),
						},
```

Reply:
It's more of a triple-value setting (there's an underlying bool which we infer the value of) so we'll probably have to stick with enum-like strings